shared_workflow/shared.py

```diff
@@ -353,10 +355,14 @@ def get_input_wc():
     return user_input_wc
 
 
-def set_wct(est_run_time, ncores, auto=False):
+def set_wct(est_run_time, ncores, auto=False, logger=get_basic_logger()):
     import estimation.estimate_wct as est
-
-    print(
+    if auto:
```

Comment:
level = DEBUG if auto else INFO (I just prefer it cos its more compact, up to you though)